### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by jgrund**
- Change lingering refs.
- use latest stable rustfmt to cleanup files.
- Update travis.yml to use distinct stages.
- Bump some deps to latest.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -54,3 +50,13 @@ jobs:
               script: ./travis_wait "./include/travis/run_in_centos7_docker.sh ./include/travis/copr-deploy.sh build"
               on:
                   all_branches: true
+stages:
+    - test
+    - name: deploy-libzfs-sys
+      if: branch =~ ^v.+libzfs-sys$
```
Comment (johnsonw) [inline .travis.yml]: Why doesn't this stage check for a number in the version like deploy-libzfs does?

File: .travis.yml [hunk 2]
```diff
@@ -54,3 +50,13 @@ jobs:
...
+    - name: deploy-libzfs
+      if: branch =~ ^v.+\dlibzfs$
+    - name: deploy-node-libzfs
+      if: branch =~ ^v.+node-libzfs$
```
Comment (johnsonw) [inline .travis.yml]: Why doesn't this branch check for a number lik deploy-libzfs does?

Comment (jgrund) [reply .travis.yml]: `deploy-libzfs` does the same thing.

I'll update these to look more like `deploy-copr`.

File: .travis.yml (hunk 1 above)
Comment (jgrund) [reply .travis.yml]: `deploy-libzfs` does the same thing.

I'll update these to look more like `deploy-copr`.

**Pull request merged**